system/http.go

```diff
@@ -86,6 +86,11 @@ func (u *DefHTTP) setup() error {
 		return u.err
 	}
 	req.Header = u.RequestHeader.Clone()
+
+	if host, ok := u.RequestHeader["Host"]; ok {
+		req.Host = host[0]
+	}
```

Comment:
Wonder if we should use the Header.Get method: https://golang.org/pkg/net/http/#Header.Get

```suggestion
	if host := req.Header.Get("Host"); host != "" {
		req.Host = host
	}
```